config/sync/core.extension.yml

```diff
@@ -115,6 +115,7 @@ module:
   shs: 0
   simplesamlphp_auth: 0
   smart_date: 0
+  sophron: 0
```

Comment:
Where is the being included?

Reply:
Why do we needs it?

Reply:
the latest imagemagick requires it https://git.drupalcode.org/project/imagemagick/-/blob/8.x-3.x/imagemagick.info.yml#L10